packages/address-book-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING:** Add `addressType` flag to address book entries ([#828](https://github.com/MetaMask/controllers/pull/828))
```

Comment:
@amerkadicE has pointed out that we can make this non-breaking by making this property optional. @amerkadicE will submit a new PR with this change soon, so that we can have one less breaking change and state migration.

Reply:
Resolved here: 35c2d1506acc2290548d3ab63c7fa8e9e3919039